synapse/storage/util/id_generators.py

```diff
@@ -258,6 +258,11 @@ def __init__(
 
         self._sequence_gen = PostgresSequenceGenerator(sequence_name)
```

Comment:
This uses `PostgresSequenceGenerator` directly (instead of `build_sequence_generator`) since you *have* to use postgresql for `MultiWriterIdGenerator` to make sense?

Reply:
Yup